research.md

```diff
@@ -7,19 +7,7 @@
 
 <h2>Publications by Year</h2>
 
-<!-- Super important: do not indent the content line, because then it -->
-<!-- gets treated like code -->
-<!-- Old stuff now, Alex has a better suggestion
-{% assign sortedyrs = (site.publications | sort: 'name') | reverse %}
-{% for pubyr in sortedyrs %}
-
-<h3>{{ pubyr.name }}</h3>
-
-{{ pubyr.content | markdownify }}
-
-{% endfor %}
--->
-{% assign sortedyrs = (site.data.publications | sort) %}
+{% assign sortedyrs = site.data.publications | sort %}
```

Comment:
Parenthesis are syntax errors in Liquid. Removing them fixed the `Liquid Warning: Expected dotdot but found pipe` build warning.